### DISCUSSION TITLE
pattern_error_handling

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/labview_book#70] [Announcements]
# pattern_error_handling

LabVIEW 编程语言

https://lv.qizhen.xyz/pattern_error_handling

**Comment by Fanhezhao** [comment]
“练习
考虑一下如何处理一下的情况。”处理一下应该是处理以下

**Reply by ruanqizhen** [reply]
已修改，多谢指正。

**Comment by ruanqizhen** [comment]
之前博客上的相关文章，搬到这里：

### 怎样根据错误代码得到错误信息

大多数 VI 都会带有错误处理机制，所以 VI 的前面板上会有 error in/error out 控件。如果发现有返回错误代码，之间在空间边缘处点击鼠标右键，选择 Explain Error 就可以看到详细的错误信息。

在 Explain Error 对话框上改变错误代码，即可查看到任意一个错误代码的相关信息。也可以通过菜单 Help->Explain Error 打开这个对话框。

 
图1：使用 Explain Error 对话框

如果是在程序当中需要动态的得到一个错误代码的信息，可以使用 [LabVIEW]\vi.lib\Utility\error.llb\Error Code Database.vi。这个 VI 可以根据输入的错误代码返回错误信息。

**Reply by ruanqizhen** [reply]
之前博客下的留言：

arrowpig
2008年07月4日 18:15 编辑
后来不是有Err Tag了吗？当时我还想设计一套成框架的Error Handling的东东呢，后来也不了了之了
